subprojects/core/src/main/java/org/gradle/api/execution/TaskOutputCacheListener.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.execution;
+
+import org.gradle.api.Task;
+
+/**
+ * Listener for events of the Task output cache
```

Comment:
I know it's OCD, but the JLS recommends writing Javadocs in full sentences, and I do miss those periods. :)